cloud-chatops/cron-clone.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# When copying the contents of this file, remember to name the new file using only valid characters [a-zA-Z0-9_-]
+# Fullstops '.' and file extenstions are not permitted
+
+set -euo pipefail -x
+
+git fetch && git clean -fxd && git reset --hard origin/main
```

Comment:
If the job is calling this script every time it runs i think you'll want to check if the repo exists locally.
If not, clone it as it should be the first time script has run and then subsequent runs it will just move on to the git fetch.

Reply:
Good suggestion but I don't think it's necessary. If the repo is not on the VM then the ChatOps stuff won't work in Slack and we will know about it. Once ChatOps has been set up, there shouldn't be any reason the repo doesn't exist.